src/main/resources/db/test-datavarehus/V0.01__dvh_init.sql

```diff
@@ -1,9 +1,23 @@
 -- Tabeller fra datavarehus
 create schema dt_p;
 
-create table dt_p.V_DIM_IA_SEKTOR (
- SEKTORKODE CHAR(1),
- SEKTORNAVN VARCHAR(60)
+create table dt_p.V_DIM_IA_SEKTOR
```

Comment:
Hva er tanken bak V0.01 i stedet for V1?

Reply:
Tenkte at "DVH" migrering er bare lokalt, til testing. Og jeg ønsket å _markere_ disse migreringene som "disse kommer ikke til prod". 
I tillegg til det kjører de først og ikke forstyrer de andre migrasjonene.